### ISSUE TITLE
unamle to upload to google play

### ISSUE THREAD

**Issue by zwieblum** [part-cw/lambdanative#459]
Using the current toolchain ist's not possible to upload an app to goole play any more.

It is possible to compie a working app for ANDROIDAPI=34 when you use
./sdkmanager "platform-tools"
./sdkmanager "build-tools;28.0.2"
./sdkmanager "ndk;26.1.10909125"
./sdkmanager "extras;android;m2repository"
./sdkmanager "platforms;android-34"

You need to convert the APK to an AAB to upload the app to goole play.

The created app (any demo app will do) works in the emulator and on real devices, but after uploading to google play the app is rejected after some days due to being not responsive.

I'm feeling like this is the end of line for the current build system :(

**Comment by mgorges** [comment]
Maybe it is time to retire [LambdaNative](https://www.lambdanative.org), which is sad, but all good things must end eventually. Was it a good idea: yes, did some valuable things come from it: also yes, but was it ever sustainable: no - the language is too niche, it was almost entirely built and maintained by our research team, and we never reached a critical mass of community members contributing to it. 

Unless @ddunsmuir has the capacity and resources to work on replacing the Android toolchain, I don't see a way forward. I don't use Android in my research program, so I can't justify dedicating time to it as this is not academically productive for me. Other parts of LambdaNative may continue (desktop-OS and iOS to some degree), but as you wrote, maybe this is the end of the line for Android and soon other pieces, too. Sorry!

**Comment by ddunsmuir** [comment]
Yes, unfortunately I don't have the resources to focus on that either at this time. As a research team we are using apps installed manually outside the app store and are not focused on publicly available apps at this time so it's unfortunately not a priority to fix this to be able to upload to the Google play store. Possibly in the future - but it might also make sense to rebuild some of our apps on a more modern platform.

**Comment by karliwalti** [comment]
I was surprised to read this sad news on an unrelated issue :-(
You can be proud that this has been running successfully for over 10 years and supported so many research projects!

**Comment by zwieblum** [comment]
Sad as it is, this is not the first project giving up on Android due to Googles hostility. https://github.com/jasonelle/jasonelle/ is the other project I was looking into, but it also dropped Android. Flutter may be a way to go, but I wonder if it is possible to make a Lambdanative_to_Flutter converter and let others handle the OS specific part? 

IMO Lamdanative was and still is a great tool and way easier to handle than all the other platform independent options.
